Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-bootstrap/2.11.3-SNAPSHOT/jspwiki-bootstrap-2.11.3-SNAPSHOT.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-bootstrap/2.11.0/jspwiki-bootstrap-2.11.0.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.bootstrap.WikiBootstrapServletContextListener  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiBootstrapServletContextListener()
	===  UNCHANGED METHOD: PUBLIC void contextDestroyed(javax.servlet.ServletContextEvent)
	===  UNCHANGED METHOD: PUBLIC void contextInitialized(javax.servlet.ServletContextEvent)
